pycon/finaid/views.py

```diff
@@ -425,3 +425,28 @@ def get_value(name, object):
         writer.writerow(data)
 
     return response
+
+
+@login_required
+def receipt_upload(request):
+    receipts = request.user.financial_aid.receipts.all()
```

Comment:
I just noticed this value isn't used on a successful POST, so we can move this line down - maybe just before the return - so we don't do an unneeded query in that case.